src/Ocean/Model/HydrostaticBoussinesqModel.jl

```diff
@@ -89,8 +89,8 @@ function vars_aux(m::HBModel, T)
     θʳ::T           # SST given    # TODO: Should be 2D
     f::T            # coriolis
     τ::T            # wind stress  # TODO: Should be 2D
-    # κ::SMatrix{3, 3, T, 9} # diffusivity tensor (for convective adjustment)
-    κᶻ::T
+    ν::SVector{3, T}
+    κ::SVector{3, T}
```

Comment:
@charleskawczynski is it possible to update the `Vars` macro so that we can write out an`SMatrix` to the vtk files? 

Ideally, these would be 3x3 matrices so that I don't have to keep calling `Diagonal` whenever I want to use them, but they aren't because then I can't output any of the auxiliary variables to the vtk files.

Reply:
Sorry for the late response. @simonbyrne was actually the master-mind behind the `Vars` type and VariableTemplates, so he's best positioned to modify this. That said, I'm pretty sure that `Vars` already uses `SMatrix`. I think for vtk export, we just need to modify [flattenednames](https://github.com/climate-machine/CLIMA/blob/master/src/Utilities/VariableTemplates/var_names.jl#L6-L9) but I'm not sure how the names/formatting plays here.